org.polymc.PolyMC.yml

```diff
@@ -0,0 +1,130 @@
+app-id: org.polymc.PolyMC
+runtime: org.kde.Platform
+runtime-version: "5.15"
+sdk: org.kde.Sdk
+command: polymc
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
Unless Java suddenly started working under Wayland, it probably doesn't make sense to include it here.

Reply:
It is working, we have included a GLFW library patch to support wayland in this package